Lib/test/test_asyncio/utils.py

```diff
@@ -546,7 +546,21 @@ def unpatch_get_running_loop(self):
 
     def setUp(self):
         self._get_running_loop = events._get_running_loop
-        events._get_running_loop = lambda: None
+
+        def _get_running_loop():
+            frame = sys._getframe(1)
+
+            if frame.f_globals['__name__'] == 'asyncio.mixins':
+                # When we called from LoopBoundedMixin we should
```

Comment:
bounded -> bound

Add a comment why this is needed. I personally have no idea and would love to learn